hedera-node/src/main/java/com/hedera/services/txns/crypto/CryptoDeleteTransitionLogic.java

```diff
@@ -87,6 +86,12 @@ public void doStateTransition() {
 				txnCtx.setStatus(result.response());
 				return;
 			}
+			var validity = ledger.usableOrElse(result.resolvedId(), INVALID_ACCOUNT_ID);
```

Comment:
may be we should give `result.response()` instead of `INVALID_ACCOUNT_ID` . Same in all places

Reply:
here `result.response()` will be `OK` right ?